_includes/plugins/katello/sidebar_3.13.html

```diff
@@ -152,6 +152,11 @@
         5.20 Content View Import/Export
       </a>
     </li>
+    <li class="item">
+      <a href="/plugins/katello/{{page.version}}/user_guide/srpm_support/index.html">
+        5.21 Source RPM Support
```

Comment:
I think our sidebar needs a complete overhaul. Maybe we can take a small step here and rather than add a new item at the end, add a sub-item to the Content item similar to how we have some items under the Advanced section.

I'd also suggest renaming this to "Source RPMs" rather than Source RPM Support which is more of the name of the goal of the feature

Reply:
I will adjust the name, after talking in irc today, it seems we can not nest it how we want it.